W-17316096 monitoring/message logging (#783)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/flex-managed-view-status.adoc

= Viewing a Managed Flex Gateway Dashboard
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

== View a Managed Flex Gateway Dashboard 

To view information about a Managed Flex Gateway Instance:

. From Anypoint Platform, select *Runtime Manager*.
. Select *Flex Gateways* in the navigation menu.

> . Select the name of the Managed Flex Gateway whose status you want to view.

Reviewer: ```suggestion
. Select the name of the Managed Flex Gateway to view its status.
```
